src/pages/index.tsx

```diff
@@ -1,82 +1,102 @@
-              <Link href="https://discord.gg/xAm2w6g">Join Discord</Link>
-            </p>
-            <h3 tw="flex items-center space-x-4">
-              <GitHub /> <span>Contributing</span>
-            </h3>
-            <p>
-              Each page footer contains an "Edit on GitHub Link". Change the
-              markdown, make a pull request, and we'll merge it! Deploys will
-              happen automagically cause the docs are hosted on Railway.
-              <br />
-              <Link href="https://github.com/railwayapp/docs">Doc Source</Link>
-            </p>
-          </section>
+        <div tw="mt-8 grid grid-cols-1 gap-8 md:grid-cols-2">
+          <OverviewLink href="reference/starters">
+            <OverviewLinkIcon>
+              <Zap size="28" tw="text-pink-800" />
+            </OverviewLinkIcon>
+            <OverviewLinkTextbox tw="">
+              <OverviewLinkHeading>Starters</OverviewLinkHeading>
+              <OverviewLinkText>See all starters you can deploy to get up and running on Railway.</OverviewLinkText>
+            </OverviewLinkTextbox>
+          </OverviewLink>
+          <OverviewLink href="deploy/builds">
+            <OverviewLinkIcon>
+              <Code size="28" tw="text-pink-800" />
+            </OverviewLinkIcon>
+            <OverviewLinkTextbox tw="">
+              <OverviewLinkHeading>Language Specific Apps</OverviewLinkHeading>
+              <OverviewLinkText>Get information on how to deploy specific applications.</OverviewLinkText>
+            </OverviewLinkTextbox>
+          </OverviewLink>
+          <OverviewLink href="develop/variables">
+            <OverviewLinkIcon>
+              <Database size="28" tw="text-pink-800" />
+            </OverviewLinkIcon>
+            <OverviewLinkTextbox tw="">
+              <OverviewLinkHeading>Variables</OverviewLinkHeading>
+              <OverviewLinkText>Understand how Railway handles variables on the platform.</OverviewLinkText>
+            </OverviewLinkTextbox>
+          </OverviewLink>
+          <OverviewLink href="reference/guides">
+            <OverviewLinkIcon>
+              <Book size="28" tw="text-pink-800" />
+            </OverviewLinkIcon>
+            <OverviewLinkTextbox tw="">
+              <OverviewLinkHeading>Guides</OverviewLinkHeading>
+              <OverviewLinkText>Tutorials on common developer scenarios.</OverviewLinkText>
+            </OverviewLinkTextbox>
+          </OverviewLink>
+        </div>
+        <div tw="grid grid-cols-1 mt-14 items-center md:grid-cols-8">
+          <div tw="bg-gray-100 rounded-md col-span-5 p-6">
+            <div tw="flex text-base font-semibold mb-4"><GitHub tw="mr-2" /> Contributing</div>
+            <div>
+              <div>Each page footer contains an "Edit on GitHub" Link. Make a pull request, and we'll merge it!</div>
+              <Link tw="flex text-pink-800 items-center mt-2" href="https://github.com/railwayapp/docs">Docs Repository <ArrowRight /></Link>
```

Comment:
This should have a hover style

![image](https://user-images.githubusercontent.com/3044853/133296104-4a91b5a5-7061-47a0-8fd7-646aeaa6d65a.png)